tests/Unit/TestModelCreator.php

```diff
@@ -8,6 +8,7 @@
 use Appleton\Taxes\Models\TaxArea;
 use Carbon\Carbon;
 use Illuminate\Support\Facades\DB;
+use SebastianBergmann\CodeCoverage\Report\Xml\Unit;
```

Comment:
ummm ...

Reply:
Wow, never seen that before. Don't know how I imported that